setup.py

```diff
@@ -61,7 +61,15 @@
     """,
     python_requires=">=3.6",
     install_requires=requires,
-    extras_require={"mpi": ["mpi4py>=1.3"], "profiling": ["psutil", "pyinstrument"]},
+    extras_require={
+        "mpi": ["mpi4py>=1.3"],
+        "compression": [
+            "bitshuffle @ git+https://github.com/kiyo-masui/bitshuffle.git",
+            "numcodecs==0.7.3",
+        ],
+        "zarr": ["zarr==2.8.1"],
```

Comment:
It does not necessarily make sense to separate `zarr` from `numcodecs`, since they are used together.

See if you can add platform specifiers, (Check for MACOS, and if MACOS, do not install `zarr`.

Reply:
numcodecs should work on macos now.